test/powershell/Modules/Microsoft.PowerShell.Utility/xml.tests.ps1

```diff
@@ -55,19 +55,31 @@
         }
 
         It "non filesystem path using <parameter>" -TestCases @(
-            @{parameter="literalPath"},
-            @{parameter="path"}
+            @{parameter="literalPath"; expectedError='ProcessingFile,Microsoft.PowerShell.Commands.SelectXmlCommand'},
```

Comment:
The original test case seems to be a positive test case, but you've changed it to a negative test case?  From the title, it seems like the test was to validate you can query xml from a non-filesystem path.  Perhaps changing this test to successfully query xml?  If a negative test cases is needed, make this a different test case.

Reply:
The original test was a negative test, testing for ProcessingFile,Microsoft.PowerShell.Commands.SelectXmlCommand

The test was not using ShouldBeErrorId correctly (misspelled and closing } in the wrong place) and actually wasn't working as expected using variable:__data.

Reply:
Can you add "..should fail" to the It description?

Reply:
Done.